### PR TITLE
Increase test coverage

### PR DESCRIPTION
**Pull request by felipeZ**
Keep on increasing the test coverage. solve #169

### CODE REVIEW

File: src/qmflows/common.py [hunk 1]
```diff
@@ -1,51 +1,49 @@
...
+    basisFormat: list
 
 
 class AtomBasisData(NamedTuple):
-    exponents: Any
-    coefficients: Any
+    """Contraction coefficients and exponents for a gaussian basis."""
 
+    exponents: np.array
+    coefficients: np.array
```
Comment (BvB93) [inline src/qmflows/common.py]: The array class is called ``np.ndarray``; ``np.array`` is a helper function for initializing the instances.

File: src/qmflows/packages/SCM.py [hunk 2]
```diff
@@ -14,11 +14,86 @@
...
 from ..type_hints import Final, WarnMap
 from ..warnings_qmflows import Key_Warning, QMFlows_Warning
-from .packages import Package, Result, get_tmpfile_name, package_properties
-
+from .packages import Package, Result, package_properties
+from ..utils import get_tmpfile_name
 # ========================= ADF ============================
 
 
+def handle_SCM_special_keywords(settings: Settings, key: str,
```
Comment (BvB93) [inline src/qmflows/packages/SCM.py]: Looks quite a bit cleaner, big fan!

File: src/qmflows/packages/orca.py [hunk 3]
```diff
@@ -74,7 +73,7 @@ def format_atom(atom: plams.Atom) -> str:
                 symbol, mass, coords = atom.symbol, atom._getmass(), atom.coords
                 return '{:2s}{:12.4f}{:14.6f}{:14.6f}{:14.6f}\n'.format(symbol, mass, *coords)
 
-            def format_hessian(dim: int, hess: List[List[float]]) -> str:
+            def format_hessian(dim, hess: Union[List[List[float]], np.array]) -> str:
```
Comment (BvB93) [inline src/qmflows/packages/orca.py]: So above; the class name is ``np.ndarray``.

File: src/qmflows/packages/orca.py [hunk 4]
```diff
@@ -111,12 +110,12 @@ def format_hessian(dim: int, hess: List[List[float]]) -> str:
...
 
             # Store the hessian in the plams_dir
-            hess_path = get_tmpfile_name()
+            hess_path = get_tmpfile_name("ORCA_hessian_")
             with open(hess_path, "w") as hess_file:
                 hess_file.write(hess_str)
 
             settings.specific.orca.geom.InHess = "read"
-            settings.specific.orca.geom.InHessName = '"' + hess_path + '"'
+            settings.specific.orca.geom.InHessName = f"{hess_path.as_posix()}"
```
Comment (BvB93) [inline src/qmflows/packages/orca.py]: Does it have to be double quotation marks (``"``) here?
If not you could also just call ``repr()``.

File: src/qmflows/packages/packages.py [hunk 5]
```diff
@@ -184,6 +183,7 @@ def get_property(self, prop: str) -> Any:
             # AMS rename all the DFTB job names
             file_pattern = "dftb.rkf"
 
+        print("plams dir: ", plams_dir)
```
Comment (BvB93) [inline src/qmflows/packages/packages.py]: Is this supposed to be here?

File: src/qmflows/parsers/orca_parser.py [hunk 6]
```diff
@@ -16,14 +16,15 @@
...
+from ..type_hints import PathLike
 
 Vector = np.ndarray
 Matrix = np.ndarray
 
 vectorize_float = np.vectorize(float)
 
 
-def parse_molecule(file_name, mol=None):
+def parse_molecule(file_name: PathLike, mol: Molecule = None) -> Molecule:
```
Comment (BvB93) [inline src/qmflows/parsers/orca_parser.py]: The  ``mol`` annotation should be ``Optional[Molecule]`` (*i.e.* ``None`` or a ``Molecule`` instance)

File: src/qmflows/common.py (hunk 1 above)
Comment (felipeZ) [reply src/qmflows/common.py]: thanks!

File: src/qmflows/packages/orca.py (hunk 4 above)
Comment (felipeZ) [reply src/qmflows/packages/orca.py]: thanks! that is a bug that I just introduced!

File: src/qmflows/packages/packages.py (hunk 5 above)
Comment (felipeZ) [reply src/qmflows/packages/packages.py]: Niet! Bedankt!

File: src/qmflows/parsers/orca_parser.py (hunk 6 above)
Comment (felipeZ) [reply src/qmflows/parsers/orca_parser.py]: indeed

**Pull request merged**